include/nlohmann/detail/conversions/from_json.hpp

```diff
@@ -325,8 +325,9 @@ inline void from_json(const BasicJsonType& j, typename BasicJsonType::binary_t&
     bin = *j.template get_ptr<const typename BasicJsonType::binary_t*>();
 }
 
-template<typename BasicJsonType, typename ConstructibleObjectType,
-         enable_if_t<is_constructible_object_type<BasicJsonType, ConstructibleObjectType>::value, int> = 0>
+template < typename BasicJsonType, typename ConstructibleObjectType,
+           enable_if_t < is_constructible_object_type<BasicJsonType, ConstructibleObjectType>::value&&
+                         !std::is_enum<typename ConstructibleObjectType::key_type>::value, int > = 0 >
```

Comment:
There's a `.clang-format` file for this repo.  You should make sure that your changes are properly formatted.

Reply:
Looks like it's already incosistent for `template<typename` vs `template < typename`.